.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,25 @@
+# New tags and releases are created by changeset https://github.com/atlassian/changesets
+name: Create a new release on GitHub when a new tag is created
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v0.4.0, v1.1.0
+
+jobs:
+  build:
+    name: Create a new release on GitHub when a new tag is created
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create release on GitHub
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          body: 'Changelog https://github.com/backstage/backstage/blob/master/CHANGELOG.md'
```

Comment:
See #3566, this one isn't used anymore. Tricky bit here is how to make a summary of the packages published in the release that led to the tag bump. We can figure out the commit based on the tag, but then we might need a script or smth that can pull together all changes into a sane summary, probably removing some of the dependency update listings as well.

Reply:
If we seek to given git tag (e.g. [`v0.4.1`](https://github.com/backstage/backstage/tree/v0.4.1), or [`v0.4.0`](https://github.com/backstage/backstage/tree/v0.4.0)), we can find the most recent merge commit, which links to a changeset PR that caused the tag to be generated. In case of `v0.4.1`, it's this Pull Request https://github.com/backstage/backstage/pull/3668

To begin with, let's use the description of the PR under `Releases` section as the release summary.

We can optimize this more, by removing "Updated Dependencies" bits.